pkgs/development/python-modules/torch-tb-profiler/default.nix

```diff
@@ -1,32 +1,31 @@
 {
+  lib,
   buildPythonPackage,
   fetchFromGitHub,
-  lib,
   pandas,
   pytestCheckHook,
-  torch,
   tensorboard,
+  torch,
   torchvision,
 }:
-
 let
-  version = "0.3.1";
+  version = "0.4.0";
   repo = fetchFromGitHub {
     owner = "pytorch";
     repo = "kineto";
-    rev = "v${version}";
-    hash = "sha256-Yg001XzOPDmz9wEP2b7Ggz/uU6x5PFzaaBeUBwWKFS0=";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-nAtqGCv8q3Tati3NOGWWLb+gXdvO3qmECeC1WG2Mt3M=";
   };
 in
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "torch_tb_profiler";
   inherit version;
-  format = "setuptools";
+  pyproject = true;
```

Comment:
dunno, it seams to work without it.

Reply:
It is probably added somewhere impure. Please add setuptools regardless so that it doesn't break in the future, it needs it https://github.com/pytorch/kineto/blob/main/tb_plugin/setup.py#L7